lib/models/Currencies.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/foundation.dart';
+
+enum Currency { SEEDS, USD }
+
+extension CurrencyExtension on Currency {
+  String get name => describeEnum(this);
+}
+
+Currency fromCurrencyName(String name) {
+  switch (name) {
+    case "SEEDS":
```

Comment:
Can it accept any currency, or only USD?

We now support any currency in general in the app, with the app setting that sets the default currency. 

It's set to the user's locale currency intiially, they can change it in settings. 

Seems like we need to support all currencies.